android/settings.gradle

```diff
@@ -1,10 +1,10 @@
 rootProject.name = 'ReactNativeBase'
-include ':react-native-i18n'
-project(':react-native-i18n').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-i18n/android')
-include ':app'
```

Comment:
Did you test if the app runs in android without this?

Reply:
@mato2593 Actually i removed that library so we don't need to specify that in gradle anymore. Is that what you meant?

Reply:
No, I meant the `include ':app'`, which is the main module of the app

Reply:
@mato2593 Oh, i move that to the end of the file... i think it's the same thing right?

Reply:
I didn't see it 😅